content/SCALE/SCALEUIReference/Credentials/Certificates/CertificatesScreensSCALE.md

```diff
@@ -134,34 +138,36 @@ The **Extra Constraints** settings change based on the selection in **Type** on
 | **Critical Extension** | Select to identify this extension as critical for the certificate. The certificate-using system must recognize the critical extensions to prevent this certificate being rejected. The certificate-using system can ignore extensions identified as not critical and still approve the certificate. |
 | **Key Usage** | Select to activate this certificate extension. The key usage extension defines the purpose (e.g., encipherment, signature, certificate signing) of the key contained in the certificate. The usage restriction might be employed when a key that can be used for more than one operation is to be restricted. For example, when an RSA key should be used only to verify signatures on objects other than public key certificates and CRLs, the Digital Signature bits are asserted. Likewise, when an RSA key should be used only for key management, the Key Encipherment bit is asserted. See [RFC 3280, section 4.2.1.3](https://www.ietf.org/rfc/rfc3280.txt) for more information. Displays the **Key Usage Config** field. |
 | **Key Usage Config** | Displays after selecting **Extended Key Usage** or **Key Usage**. Select the option that specifies valid key usages for this certificate. Options are **Digital Signature**, **Content Commitment**, **Key Encipherment**, **Data Encipherment**, **Key Agreement**, **Key Cert Sign**, **CRL Sign**, **Encipher Only**, **Decipher Only** or **Critical Extension**. Web certificates typically need at least **Digital Signature** and possibly **Key Encipherment** or **Key Agreement**, while other applications might need other usages.  |
 {{< /truetable >}}
 {{< /expand >}}
 
-#### Extra Constraints - Import Certificate
-When **Type** on **Identifier and Type** is set to **Import Certificate** the **Extra Constraints** screen does not include the options to set extension types.
+#### Import Certificate Options
+When **Type** on **Identifier and Type** is set to **Import Certificate** the **Import Certificate** options screen displays.
 {{< expand "Click Here for More Information" "v" >}}
 
-![AddCertificateImportExtraConstraints](/images/SCALE/22.02/AddCertificateImportExtraConstraints.png "Add Certificate Extra Constraints Import Certificate") 
+![AddCertificateImportOptions](/images/SCALE/23.10/AddCertificateImport.png "Add Certificate Import Certificate Options")
 
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
 | **Certificate** | Required. Paste the certificate for the CA into this field. |
+| **CSR exists on this system** | Check this box if importing a certificate for which a CSR exists on this system. Displays the **Certificate Signing Request** dropdown. |
```

Comment:
When talking about a checkbox, just use "select" and no need to mention the checkbox.
```suggestion
| **CSR exists on this system** | Select if importing a certificate for which a CSR exists on this system. Displays the **Certificate Signing Request** dropdown. |
```